templates/ntp.conf.erb

```diff
@@ -31,7 +31,7 @@ interface listen <%= interface %>
 server <%= server %><% if @iburst_enable == true -%> iburst<% end %><% if @preferred_servers.include?(server) -%> prefer<% end %>
 <% end -%>
 
-<% if scope.lookupvar('::is_virtual') == "false" or @udlc -%>
+<% if scope.lookupvar('::is_virtual') == "true" or @udlc -%>
```

Comment:
why does this use scope.lookupvar anyway?
are facts not exposed as `@fact`, too?